Vaccine/src/main/java/dk/sdu/se/f23/InVasion/vaccine/VaccineControlSystem.java

```diff
@@ -20,13 +20,14 @@
 import java.util.List;
 
 public class VaccineControlSystem implements EntityProcessingService, EventListener {
-    private List<Entity> targets = new ArrayList<>();
+    private final List<Entity> targets = new ArrayList<>();
```

Comment:
```suggestion
    private final List<Entity> TARGETS = new ArrayList<>();
```

Reply:
This is not convention since it is a List and the content of the list can change